src/averageAges.js

```diff
@@ -15,11 +25,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const calculateCentury = num => Math.ceil(num / 100);
```

Comment:
this function is alright, but it can do more. below you take the result of the function and compare it to the century and then there's a logical and. it's quite a lot.